design/run_and_collect_data.md

```diff
@@ -0,0 +1,27 @@
+#run.py proposal
```

Comment:
I did have a format - but for some reason git didnt track/add the folder it is in, so I'll be looking into why